### PR TITLE
MOSART documentation

### PR DESCRIPTION
**Pull request by hydrotian**
Adds MOSART documentation.

[BFB]

### CODE REVIEW

File: components/mosart/docs/tech-guide/index.md [hunk 1]
```diff
@@ -1 +1,38 @@
-start of the MOSART Technical Guide
+# MOSART Tech Guide
+
+This Tech Guide includes some technical details about MOSART.
+
+## Table of Conents
```
Comment (rljacob) [inline components/mosart/docs/tech-guide/index.md:5]: Please don't add an explicit table of contents in the markdown files.  A TOC is generated automatically from the headings and displayed on the right-hand side of the page.

**Review by rljacob — CHANGES_REQUESTED** [review]

File: components/mosart/docs/tech-guide/index.md (hunk 1 above)
Comment (hydrotian) [reply components/mosart/docs/tech-guide/index.md]: Done!

File: components/mosart/docs/tech-guide/index.md [hunk 2]
```diff
@@ -1 +1,38 @@
...
+
+## Physics
+
+- to be added
+
+## Parameters
+
+The following table includes the main parameters required in the MOSART parameter file.
+
+| Variable Name     | Description [unit]                                                             |
```
[the remark this answers is not in the document]
Comment (hydrotian) [reply components/mosart/docs/tech-guide/index.md:23]: We will expand the table in the future to include more parameters required by additional MOSART features.

**Review by rljacob — APPROVED** [review]

File: components/mosart/docs/user-guide/index.md [hunk 3]
```diff
@@ -1 +1,132 @@
...
+
+  - Note if `opt_eleprof` is set to `1`, the elevation profile data must be included in the MOSART parameter file.
+
+### Sediment transport
+
+- If sediment feature is activated, D50 data must be included in the MOSART parameter file. In addition, `rof_sed = .true.` has to be defined in `./user_nl_cpl` to allow sediment flux passing into the river model through the coupler.
+
+### Heat transport
+
+- to be added
```
Comment (bishtgautam) [inline components/mosart/docs/user-guide/index.md:132]: Let's remove this for now.

**Comment by bishtgautam** [comment]
@rljacob Should we squash the commits to remove multiple commits that fix the formatting errors?

@hydrotian If @rljacob agrees with squashing the commits, do you want to squash the commits or would you like me to take care of it?

**Comment by hydrotian** [comment]
> @hydrotian If @rljacob agrees with squashing the commits, do you want to squash the commits or would you like me to take care of it?

@bishtgautam Sure, please take care of it. Thanks.

**Comment by rljacob** [comment]
Yes please squash those commits.

**Pull request merged**